infra/core/search/search-services.bicep

```diff
@@ -18,23 +18,25 @@ param encryptionWithCmk object = {
   'highDensity'
 ])
 param hostingMode string = 'default'
-param networkRuleSet object = {
-  bypass: 'None'
-  ipRules: []
-}
 param partitionCount int = 1
-@allowed([
-  'enabled'
-  'disabled'
-])
-param publicNetworkAccess string = 'enabled'
 param replicaCount int = 1
 @allowed([
   'disabled'
   'free'
   'standard'
 ])
 param semanticSearch string = 'disabled'
+@allowed(['Enabled', 'Disabled'])
```

Comment:
I believe this has to be all lower:

https://learn.microsoft.com/en-us/azure/templates/microsoft.search/searchservices?pivots=deployment-language-bicep#searchserviceproperties

Reply:
We ran into an issue when using upper.  So , please test it.

Reply:
Re upper: my tests currently seem like it's working to use 'Enabled' vs 'enabled' but I haven't tested 'Disabled' (VNet with no IP). @tonybaloney It looks like you changed it from 'enabled' to 'Enabled' for this file, any particular reason why? It seems safer to go with whats on the Bicep page, but am also nervous since there are quite a few instances of 'Enabled'/'Disabled' and am not sure if they should all be lowercase.

Reply:
All the other ARM specifications have `Enabled`, `Disabled`. For some reason, the search API is lower case. https://learn.microsoft.com/en-us/azure/templates/microsoft.search/searchservices?pivots=deployment-language-bicep#searchserviceproperties 
I've added a ternary case to lower-case them just for this instance so that the bicep linter doesn't complain.

Reply:
Yes, search is special.